<?xml version="1.0" ?>
<launch>
  <node name="external_force_jump_detector" pkg="sven_ros" type="external_force_jump_detector_node" output="screen" required="true">
    <rosparam file="$(find sven_ros)/config/jump_detector/external_force.yaml" command="load"/>
  </node>
</launch>
